src/style.css

```diff
@@ -0,0 +1,171 @@
+:root {
+  --main-accent: #060b35;
+  --secondary-color: #616070;
+  --white: #fff;
+  --blue-accent: #00acdc;
+  --gray-elements: #e5e5e5;
+  --yellow-stars-color: #ffde6a;
+  --card-padding: 16px;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: var(--main-accent);
+}
+
+body {
+  margin: 0;
+}
+
+div {
+  margin: 0;
+}
+
+p {
+  display: flex;
+  margin: 0;
+}
+
+.card {
+  justify-content: center;
+  align-items: center;
+
+  max-width: calc(198px - (var(--card-padding) * 2));
```

Comment:
this seems to be overcomplicated

Reply:
max width 200px: minus border = 198 - paddings from both sides